scipy/ndimage/tests/test_filters.py

```diff
@@ -2016,6 +2016,38 @@ def test_rank15(self, dtype, xp):
                                      origin=[-1, 0])
         assert_array_almost_equal(expected, output)
 
+    def test_rank16(self):
+        array = [3, 2, 5, 1, 4]
+        expected = np.asarray([3, 3, 2, 4, 4])
```

Comment:
Please add a comment that this test compares the output of the 1D fast path to the generic implementation from previous scipy versions, scipy <=1.14. Or if you can explain why this is the right result in a way that can be immediately verified with paper and pencil, even better.

Reply:
This is the same values everywhere. Originally taken from test_rank03 written by... you? It can be immediately verified with paper and pencil...

Reply:
This is by definition:
- median(3,2,5) = 3
- median( 2, 5, 1) = 2
etc.
IMO, it is too long for a comment as there are 5 such calculations. Since it is by definition, the reader is expected to know it.

Reply:
There exists at least one user for whom it was not obvious immediately (me), so I added a small comment to this effect.